Packs/FeedAlienVault/Integrations/FeedAlienVaultOTXTaxii/FeedAlienVaultOTXTaxii.py

```diff
@@ -0,0 +1,667 @@
+        """
+        return StixDecode.decode(response)
+
+
+def module_test_command(client: Client, args: Dict):
+    """Test module for the integration
+    will run on all the collections given and check for a response.
+    if all_collections is checked will return an error only in case no collection returned a response.
+    if all_collections is not checked will return an error for the collections that did not respond.
+
+    Args:
+        client(Client): The AlienVault OTX client.
+        args(dict): empty dictionary.
+
+    Returns:
+        str,dict,dict. ok if passed - will raise an Exception otherwise.
+    """
+    passed_collections = []  # type:List
+    failed_collections = []  # type:List
+    for collection in client.collections:
+        try:
+
+            client.build_iterator(collection)
+            passed_collections.append(collection)
+
+        except Exception as e:
+            if e.__class__ is requests.exceptions.SSLError:
+                raise Exception("SSL Connection failed - try marking the Trust Any Certificate checkbox.")
+            else:
+                if not client.all_collections:
+                    failed_collections.append(collection)
+                    continue
+
+    if not client.all_collections and len(failed_collections) > 0:
+        raise Exception(f"Unable to poll from the collections {str(failed_collections)} check the collection names and "
+                        f"configuration on Alien Vault")
+
+    if len(passed_collections) == 0:
+        raise Exception("Unable to poll from any collection - please check the configuration on Alien Vault")
+
+    return 'ok', {}, {}
+
+
+def get_indicators_command(client: Client, args: Dict):
+    """Runs fetch indicators and return the indicators.
+
+    Args:
+        client(Client): The AlienVault OTX client.
+        args(dict): The command arguments
+
+    Returns:
+        str,dict,dict. The human readable, and rawJSON from the command - no context created.
+    """
+    limit = int(args.get('limit', 50))
+    indicator_list = fetch_indicators_command(client, limit)
+
+    human_readable = tableToMarkdown("Indicators from AlienVault OTX TAXII:", indicator_list,
+                                     headers=['value', 'type'], removeNull=True)
+
+    return human_readable, {}, indicator_list
+
+
+def parse_indicators(sub_indicator_list, full_indicator_list):
+    """Gets a decoded indicator list and returns a parsed version of the indicator with accordance with Demisto's
+    Feed indicator standards.
+
+    Args:
+        sub_indicator_list(list): A list of STIXDecoded indicators
+        full_indicator_list(list): A list of all the indicators fetched to this point - used to prevent duplications.
+
+    Returns:
+        list,list. A list of parsed indicators and an updated list of all indicators polled
+    """
+    parsed_indicator_list = []  # type: List
+    for indicator in sub_indicator_list:
```

Comment:
seems like you abusing the indicator here, you should create a new object which will hold the real indicator within the rawJson, the value and the type, the last two should be in the outer scope as well.
Oh and the fields too, are we sure that it's all the mapping we want to do?

Reply:
The value should also be part of the rawJSON

Reply:
done